src/tools/tests/GenerationBehaviorTest.kt

```diff
@@ -0,0 +1,53 @@
+package src.tools.tests
+
+import arcs.core.data.RawEntity
+import arcs.core.data.Schema
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class GenerationBehaviorTest {
```

Comment:
Should I create equivalent tests that run in a TS / Wasm environment?

Reply:
Step by step, I think this is ok for now. I don't think we should actively invest in WASM given lack of customers right now.